.ci/full-downstream-config.yaml

```diff
@@ -6,7 +6,7 @@ default:
   build-command:
     current: mvn -e -nsu --builder smart -T1C clean install -Dfull -DskipTests -Dmaven.wagon.httpconnectionManager.ttlSeconds=25 -Dmaven.wagon.http.retryHandler.count=3
     upstream: mvn -e --builder smart -T1C clean install -DskipTests -Dgwt.compiler.skip=true -Dgwt.skipCompilation=true -Denforcer.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true -Drevapi.skip=true -Dmaven.wagon.httpconnectionManager.ttlSeconds=25 -Dmaven.wagon.http.retryHandler.count=3
-    downstream: mvn -e -nsu clean install -Dfull -Pbusiness-central,wildfly,sourcemaps,no-showcase -Dcontainer=wildfly -Dcontainer.profile=wildfly -Dintegration-tests=true -Dcargo.ignore.failures=true -Dmaven.test.failure.ignore=true -Dmaven.test.redirectTestOutputToFile=true -Dgwt.compiler.localWorkers=1 -Dwebdriver.firefox.bin=/opt/tools/firefox-60esr/firefox-bin -Dgwt.skipCompilation=true -Dmaven.wagon.httpconnectionManager.ttlSeconds=25 -Dmaven.wagon.http.retryHandler.count=3
+    downstream: mvn -e -nsu clean install -Dfull -Pbusiness-central,wildfly,sourcemaps,no-showcase -Dcontainer=wildfly -Dcontainer.profile=wildfly -Dintegration-tests=true -Dmaven.test.redirectTestOutputToFile=true -Dgwt.compiler.localWorkers=1 -Dwebdriver.firefox.bin=/opt/tools/firefox-60esr/firefox-bin -Dgwt.skipCompilation=true -Dmaven.wagon.httpconnectionManager.ttlSeconds=25 -Dmaven.wagon.http.retryHandler.count=3
```

Comment:
@danielezonca  `-Dcargo.ignore.failures=true -Dmaven.test.failure.ignore=true` doesn't disable the tests but only immediate failure due them. We need to continue until the end of build to show ALL failures/errors in tests. If there is compilation error or any other build error it fails fast.

See http://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html and

Reply:
If this is on purpose it is still not working properly: for example check this PR
https://github.com/kiegroup/drools/pull/3527
There was a clearly broken test (just throwing `new UnsupportedOperationException();`)
https://github.com/kiegroup/drools/pull/3527#discussion_r621038604
that was not making fail the corresponding GHA builds (only Jenkins build is correctly reporting the error)
https://github.com/kiegroup/drools/pull/3527/checks?sha=292cc5a465d3f511b53c0bf160da0b8770a71ae5
![image](https://user-images.githubusercontent.com/244607/116879649-2f284a80-ac21-11eb-93d1-0d8578163827.png)

If the idea is to execute also downstream projects before reporting failure the configuration is probably fully working 🤔 

Btw as side note I'm not sure if it is useful to build downstream repos if there is a broken test in the current one because the final error could be quite misleading: let assume a PR breaks Drools badly, of course also all other downstream repos will fail but the _real_ error is just the first failure.
I agree that we don't want report only the first failure but I would limit this to the current repo with `-fae` option and not proceed also with downstream repos. Wdyt?

Reply:
I agree that if GA is not taking that as failure we need to fix that. Could you please @Ginxo try to ask support or find out why this happens? not respecting the failure status in build-chain tool extension or forget that if next repository is in success?

With regard the side note to if we want to have that, surely we DO. As you have more than 10 repos in downstream list, if you remove that option you will see only the last failed repository and if you don't test it locally the same way like Jenkins/GA you will need more iteration to see green. I agree that this is transferred responsibility or trust from individual contributors to our CI which can help with complex downstream workflow, but still if we remove that option/s/ we will get more unsatisfied results/complaints I think.